package.json

```diff
@@ -14,7 +14,7 @@
     "prettier": "prettier --write 'src/**/*.sol' 'test/**/*.sol' 'package/**/*.ts' 'script/**/*.ts' 'wagmi.config.ts'",
     "coverage": "forge coverage --report lcov",
     "write-gas-report": "forge test --gas-report > gasreport.ansi",
-    "prepack": "yarn wagmi && yarn bundle-configs && yarn build",
+    "prepack": "yarn wagmi && forge build && cp out/**/*.abi.json artifacts/ && yarn bundle-configs && yarn build",
```

Comment:
instead of copying the files to another location, how about just listing the file folder in `/files` in the `package.json`:

(package.json/files)[https://github.com/ourzora/zora-1155-contracts/blob/f79a115950af8122a3fda247624948e5f4354755/package.json#L31l]

Reply:
I want to move them to the root level and only grab out the abi files. Can you do a file glob in files?